docs/apache-airflow/start/docker-compose.yaml

```diff
@@ -146,13 +146,54 @@ services:
 
   airflow-init:
     <<: *airflow-common
-    command: version
+    command:
+      - bash
+      - -c
+      - |
+        if [[ -z "${AIRFLOW_UID}" ]]; then
+          echo -e "\033[1;31mERROR!!!: AIRFLOW_UID not set!\e[0m"
+          echo "Please follow these instructions to set AIRFLOW_UID And AIRFLOW_GID:
```

Comment:
```suggestion
          echo "Please follow these instructions to set AIRFLOW_UID and AIRFLOW_GID:
```

Reply:
It's explaineid in the instructions, ut yeah - no harm, as longas the line will not get too long.